backend/libraries/chat_events/src/chat_events.rs

```diff
@@ -1862,9 +1861,9 @@ impl ChatEvents {
                         });
                     }
                     if let MessageContentInternal::Prize(mut p) = m.content {
-                        if let Some(refund) = p.prize_refund(m.sender, &MEMO_PRIZE_REFUND, now * 1_000_000) {
-                            result.prize_refunds.push(refund);
-                        }
+                        result
+                            .final_prize_payments
+                            .append(&mut p.final_payments(m.sender, now * 1_000_000));
```

Comment:
I think `.extend(p.final_payments(m.sender, now * 1_000_000));` would be the more idiomatic way of doing this.